<ui version="4.0" >
 <class>FormListaComprasBase</class>
 <widget class="QWidget" name="FormListaComprasBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="statusTip" >
   <string/>
  </property>
  <property name="whatsThis" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:11pt; font-weight:600;">Listado de Compras&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="GBDatos" >
         <property name="title" >
          <string>Historial de compras</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QTableView" name="TVCompras" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GBFiltrado" >
         <property name="title" >
          <string>&amp;Filtrado</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="ChBProveedor" >
              <property name="text" >
               <string>Proveedor:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBProveedor" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="ChBFecha" >
              <property name="text" >
               <string>Fecha:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="DEfecha" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QCheckBox" name="ChBProducto" >
              <property name="text" >
               <string>Producto:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBProducto" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QPushButton" name="PBAgregar" >
         <property name="text" >
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBBorrar" >
         <property name="text" >
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBProductos" >
         <property name="text" >
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="PBCerrar" >
         <property name="text" >
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TVCompras</tabstop>
  <tabstop>PBAgregar</tabstop>
  <tabstop>PBBorrar</tabstop>
  <tabstop>PBProductos</tabstop>
  <tabstop>GBFiltrado</tabstop>
  <tabstop>ChBProveedor</tabstop>
  <tabstop>CBProveedor</tabstop>
  <tabstop>ChBFecha</tabstop>
  <tabstop>DEfecha</tabstop>
  <tabstop>ChBProducto</tabstop>
  <tabstop>CBProducto</tabstop>
  <tabstop>PBCerrar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
